src/ui/server/helpers/is-policy-contact-data-same-as-owner/index.ts

```diff
@@ -0,0 +1,54 @@
+import { ApplicationPolicyContact, ApplicationOwner } from '../../../types';
+import ACCOUNT_FIELD_IDS from '../../constants/field-ids/insurance/account';
+import { isPopulatedString } from '../string';
+
+const { FIRST_NAME, LAST_NAME, EMAIL } = ACCOUNT_FIELD_IDS;
+
+/**
+ * stringsAreDefined
+ * checks that strings are not null or undefined and greater length than 0
+ * @param {String} ownerString - string for firstName, lastName or email
+ * @param {String} policyContactString - string for firstName, lastName or email
+ * @returns {Boolean}
+ */
+const stringsAreDefined = (ownerString: string, policyContactString: string) => isPopulatedString(ownerString) && isPopulatedString(policyContactString);
+
+/**
+ * stringsAreEqual
+ * checks that strings are the same
+ * @param {String} ownerString - string for firstName, lastName or email
+ * @param {String} policyContactString - string for firstName, lastName or email
+ * @returns {Boolean}
+ */
+const stringsAreEqual = (ownerString: string, policyContactString: string) => ownerString === policyContactString;
+
+/**
+ * isPolicyContactDataSameAsOwner
+ * checks if policyContact data for names and email is same as application owner
+ * returns true if all are the same
+ * @param {ApplicationOwner} owner - application owner
+ * @param {ApplicationPolicyContact} policyContact - policyContact
+ * @returns {Boolean}
+ */
```

Comment:
Can we move the above string functions into their own helper files please, to be generic? Params can be e.g `str1` `str2` rather than owner/contact specifics